Doc/whatsnew/3.14.rst

```diff
@@ -771,6 +771,92 @@ asyncio
 * Removed implicit creation of event loop by :func:`asyncio.get_event_loop`.
   It now raises a :exc:`RuntimeError` if there is no current event loop.
   (Contributed by Kumar Aditya in :gh:`126353`.)
+  There's a few patterns that use :func:`asyncio.get_event_loop`, most
+  of them can be replaced with :func:`asyncio.run`.
+
+  If you're running an async function, simply use :func:`asyncio.run`,
+  before::
+
+    async def main():
+        ...
+
+    loop = asyncio.get_event_loop()
+    try:
+       loop.run_until_complete(amain())
+    finally:
+       loop.close()
+
+  after::
+
+    async def main():
+        ...
+
+    asyncio.run(main())
+
+  If you need to start something, e.g. a server listening on a socket
+  and then run forever, use :func:`asyncio.run` and an
+  :class:`asyncio.Event`, before::
+
+    def start_server(loop):
+        ...
+
+    loop = asyncio.get_event_loop()
+    try:
+        start_server(loop)
+        loop.run_forever()
+    finally:
+        loop.close()
+
+  after::
+
+    def start_server(loop):
+        ...
+
+    async def amain():
```

Comment:
Why switch to `amain()` instead of `main()` like before? I think all examples should use a consistent style.

```suggestion
    async def main():
```

Reply:
Yeah I prefer the amain, it's compatible with idiomatic usage of console scripts:

```python
import asyncio
import sys

async def amain():
    ...

def main():
    return asyncio.run(amain())

if __name__ == "__main__":
    sys.exit(main())
```